book/src/concepts/entity_and_component.md

```diff
@@ -0,0 +1,105 @@
+# Entity and Component
+
+## What are `Entity` and `Component` ?
+
+An `Entity` conceptually represents a single object in your world. `Component` represents one aspect of an object. For example, a bottle of water has a shape, a volume, a color and is made of a material (usually plastic). In this example, the bottle is the entity, and the properties are components.
+
+## Entity and Component in Amethyst
+In an inheritance design, entity usually contains components. All the data and methods related to an entity are stored within. However, in the ECS design, entity is just a general purpose object. In fact, the implementation of `Entity` in Amethyst is simply:
+```rust,no_run,noplaypen
+struct Entity(u32, Generation);
+```
+where u32 is the id of the entity and generation is used to check if the entity has been deleted.
+
```

Comment:
Is generation used to check if an entity is deleted @torkleyy ?

Reply:
I copied this line (and only this line) from specs :)